opam

```diff
@@ -1,6 +1,6 @@
 opam-version: "1.2"
 name: "ppx_deriving_cmdliner"
-version: "0.4.0"
+version: "0.4.1"
```

Comment:
While not-released it's better to call versions with a `-dev` suffix: `0.4.1-dev`